app/lib/otis/jira_client/registration.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+require "jira-ruby"
+
+module Otis
+  class JiraClient::Registration < JiraClient
+    attr_reader :registration, :finalize_url
+
+    # "Elevated access registration" issuetype
+    EA_REGISTRATION_ISSUETYPE_ID = "10715"
```

Comment:
Should these be pulled out to a separate configuration file, or are they likely to be static enough this is OK?

Reply:
Thus far they seem to be pretty static, perhaps more than I expected. I would rather have the declaration and use all in one place for now.